src/RawData/storageController.ts

```diff
@@ -67,24 +72,24 @@ export const readRawDataParameter = async (
 };
 
 export const processRawDataFile = async (fileId: string) => {
-  const filepath = path.join(
-    process.env.DATA_PATH,
-    UNPROCESSED_FOLDER,
-    fileId + ".json"
-  );
+  const filepath = path.join(process.env.DATA_PATH, UNPROCESSED_FOLDER, fileId);
   console.log("Processing", filepath);
 
   await mkdir(path.join(process.env.DATA_PATH, PROCESSED_FOLDER, fileId), {
     recursive: true
   });
 
-  const buffer = await readFile(filepath);
-  const bufferStream = new stream.PassThrough();
-  bufferStream.end(buffer);
+  const fileStream = new stream.PassThrough();
+  const chunks = await readdir(filepath);
+  for (const chunk of chunks) {
+    const buffer = await readFile(path.join(filepath, chunk));
+    fileStream.write(buffer);
+  }
+  fileStream.end();
 
   await new Promise((resolve, reject) => {
     const outstandingFunctions: (() => {})[] = [];
-    bufferStream
+    fileStream
       .pipe(JSONStream.parse(["posterior", "content", { emitKey: true }]))
       .on("data", async (data: { key: string; value: any[] }) => {
         // TODO: Pause stream until processed
```

Comment:
Has this TODO been done yet, or should it be removed?

Reply:
It hasn't been implemented. Can be addressed in a later PR.

Reply:
Cool